dependency-check-suppress-es.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These have fixed in 6.8 versions but it is recommend to not use a 6.8 client against a 6.7 server (which we have in AWS)